### PR TITLE
update commands to new version

### PR DESCRIPTION
**Pull request by AugustoPeres**

### CODE REVIEW

**Review by IvanPombo — CHANGES_REQUESTED** [review]
You also need to change in the `lib/scenarios.py`. Then provide proof of working, and we can give a greenlight!

**Review by IvanPombo — CHANGES_REQUESTED** [review]
Address also the docs.

**Comment by AugustoPeres** [comment]
Screenshot 2024-01-24 at 17 03 21
have not tested 2 because first need to install ffmpeg.
Screenshot 2024-01-24 at 17 19 38

**Comment by IvanPombo** [comment]
> Screenshot 2024-01-24 at 17 03 21 have not tested 2 because first need to install ffmpeg. Screenshot 2024-01-24 at 17 19 38

The doc `docs/1_SPH_SIMULATIONS.md` also needs to be updated.

File: docs/1_SPH_SIMULATIONS.md [hunk 1]
```diff
@@ -115,12 +115,10 @@ $ cat simple_dualsphysics.py
...
 commands = [
-    {"cmd": "gencase flow_cylinder flow_cylinder -save:all", "prompts": []},
-    {"cmd": "dualsphysics flow_cylinder flow_cylinder -dirdataout data -svres",
-     "prompts":[]},
-    {"cmd": "partvtk -dirin flow_cylinder/data -savevtk vtk/PartFluid " \
-            "-onlytype:+fluid",
-     "prompts": []}]
+    "gencase flow_cylinder flow_cylinder -save:all",
+    "dualsphysics flow_cylinder flow_cylinder -dirdataout data -svres",
+    "partvtk -dirin flow_cylinder/data -savevtk vtk/PartFluid "-onlytype:+fluid"
```
Comment (IvanPombo) [inline docs/1_SPH_SIMULATIONS.md]: Typo, extra "..

```suggestion
    "partvtk -dirin flow_cylinder/data -savevtk vtk/PartFluid -onlytype:+fluid"
```

**Review by IvanPombo — APPROVED** [review]
LGTM!

**Comment by AugustoPeres** [comment]
Could you also run the scripts from your side?

**Comment by IvanPombo** [comment]
> Could you also run the scripts from your side?

@AugustoPeres this is also tested and ready to go!

**Pull request merged**